doc/sphinx/changes.rst

```diff
@@ -14,7 +14,85 @@ Version 8.15
+    features such as hints, coercions, and notations.
+
+  - A :ref:`visual Ltac debugger <815LtacDebugger>` is now available in CoqIDE.
+
+See the `Changes in 8.15.0`_ section below for the detailed list of changes,
+including potentially breaking changes marked with **Changed**.
+Coq's `reference manual for 8.15 <https://coq.github.io/doc/v8.15/refman>`_,
+`documentation of the 8.15 standard library <https://coq.github.io/doc/v8.15/stdlib>`_
+and `developer documentation of the 8.15 ML API <https://coq.github.io/doc/v8.15/api>`_
+are also available.
+
+Emilio Jesús Gallego Arias, Gaëtan Gilbert, Michael
+Soegtrop and Théo Zimmermann worked on maintaining and improving the
+continuous integration system and package building infrastructure.
+
+Erik Martin-Dorel has maintained the `Coq Docker images
+<https://hub.docker.com/r/coqorg/coq>`_ that are used in many Coq
+projects for continuous integration.
+
+The OPAM repository for Coq packages has been maintained by
+Guillaume Claret, Karl Palmskog, Matthieu Sozeau and Enrico Tassi with
+contributions from many users. A list of packages is available at
+https://coq.inria.fr/opam/www/.
+
+The `Coq Platform <https://github.com/coq/platform>`_ has been maintained
+by Michael Soegtrop and Enrico Tassi.
+
+Our current maintainers are Yves Bertot, Frédéric Besson, Ali Caglayan, Tej
+Chajed, Cyril Cohen, Pierre Corbineau, Pierre Courtieu, Maxime Dénès,
+Jim Fehrle, Julien Forest, Emilio Jesús Gallego Arias, Gaëtan Gilbert,
+Georges Gonthier, Benjamin Grégoire, Jason Gross, Hugo Herbelin,
+Vincent Laporte, Olivier Laurent, Assia Mahboubi, Kenji Maillard,
+Guillaume Melquiond, Pierre-Marie Pédrot, Clément Pit-Claudel, Pierre Roux,
+Kazuhiko Sakaguchi, Vincent Semeria, Michael Soegtrop, Arnaud Spiwack,
+Matthieu Sozeau, Enrico Tassi, Laurent Théry, Anton Trunov, Li-yao Xia
+and Théo Zimmermann. See the `Coq Team face book <https://coq.inria.fr/coq-team.html>`_
+page for more details.
+
+The 41 contributors to this version are
+Tanaka Akira, Frédéric Besson, Juan C, Ali Caglayan, Cyril Cohen, Columbus240, Adrian Dapprich, Maxime Dénès,
+Christian Doczkal, Andrej Dudenhefner, Jim Fehrle, Attila Gáspár, Gaëtan Gilbert, Jason Gross,
+Hugo Herbelin, Jasper Hugunin, Bart Jacobs, Emilio Jesús Gallego Arias, Ralf Jung, Grant Jurgensen,
+Jan-Oliver Kaiser, Wojciech Karpiel, Fabian Kunze, Olivier Laurent, Yishuai Li, Erik Martin-Dorel,
+Guillaume Melquiond, Jean-Francois Monin, Pierre-Marie Pédrot, Rudy Peterson, Clément Pit-Claudel,
+Seth Poulsen, Pierre Roux, Takafumi Saikawa, Kazuhiko Sakaguchi, Michael Soegtrop, Matthieu Sozeau,
+Enrico Tassi, Laurent Théry, Anton Trunov and Théo Zimmerman.
+
+The Coq community at large helped improve the design of this new version via
+the GitHub issue and pull request system, the Coq development mailing list
+coqdev@inria.fr, the coq-club@inria.fr mailing list, the `Discourse forum
+<https://coq.discourse.group/>`_ and the `Coq Zulip chat <https://coq.zulipchat.com>`_.
```

Comment:
It seems sphinx is not happy about other instances of this link being http

Reply:
Darn